cps/templates/login.html

```diff
@@ -21,6 +21,17 @@ <h2 style="margin-top: 0">{{_('Login')}}</h2>
     {% if remote_login %}
     <a href="{{url_for('remote_login')}}" class="pull-right">{{_('Log in with magic link')}}</a>
     {% endif %}
+    <a href="{{url_for('github_login')}}" class="pull-right">
```

Comment:
We may want to pass in a variable to this page to determine if the GitHub and/or Google icons should be shown for login.  Actually, we may want to pass in a general OAuth configured variable to determine if any additional icons be shown and then individual flags for each service?

Reply:
We can use the config

```
config.config_use_github_oauth
```

and

```
config.config_use_google_oauth
```